src/frontend/src/post.tsx

```diff
@@ -122,10 +122,18 @@ export const PostView = ({
 
     React.useEffect(() => {
         const article: any = refArticle.current;
-        if (!prime && article && article.scrollHeight > article.clientHeight) {
-            article.classList.add("overflowing");
-            setShowExpandButton(true);
-        }
+        // We delay shortening the post by half a second in the hope that images would
+        // have loaded by that time. But it should be improved in the future.
+        setTimeout(() => {
```

Comment:
Could it cause the annoying effect of moving re-arranging posts while the user is looking at them? If so, then I am not sure this is a good change, since it gives an impression of a slow website.

Reply:
Yes, it can fore the upper posts. However, this is already happening when images get loaded because they are stretched and Taggr does not know in advance the exact rendered height. Also I am not sure what's worse - cutting the post in a wrong way, or what you describe? :D

Reply:
Btw, two more points:

- The effects you are afraid of will only happen on certain posts, not on all of them (those which have images and they are too long). These posts have the effect you describe already as I described above. 
- There is one way I wanted to experiment with: propagating a callback all the way to images which they would call notifying about a load. But this could be a much heavier change which I wanted to try later (as stated in comments).

Reply:
I thought a bit more and decided not to put this change into the release, because it can be see as controversial.

Reply:
Postponing sgtm. Let's experiment more here. Maybe we find a better way to improve it.